src/layouts/BlogPost.astro

```diff
@@ -64,6 +64,7 @@ const { title, description, pubDate, updatedDate, heroImage } = Astro.props;
 				<div class="prose">
 					<div class="title">
 						<div class="date">
+							By {authour} on
```

Comment:
Please handle fallback case in case there is no author, also I hope you have checked how it looks